docs/learn/encyclopedia/errors-and-debugging/debugging-errors.mdx

```diff
@@ -94,11 +94,11 @@ Note: $OPERATION here refers to any one of the Soroban operations: `INVOKE_HOST_
+| `$OPERATION_INSUFFICIENT_REFUNDABLE_FEE` | The refundable resource fee was not sufficient to cover the consumed refundable resources, i.e., not enough to pay for emitted events or TTL extensions. | The simplest fix is to increase the resource fee unconditionally; unspent fees are refunded. [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) estimates refundable fees but can't predict TTL extensions' actual cost due to ledger state volatility. Thus [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) can’t always predict the fee that will actually be charged. |
+| `INVOKE_HOST_FUNCTION_ENTRY_ARCHIVED` | Transaction tries to access an archived ledger entry, e.g. when the footprint contains a ledger key for an archived persistent Soroban ledger entry. Note, this failure happens before the Soroban host is even created, so this is unrelated to contract logic. | Restore the archived entry using RestoreFootprintOp operation. [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) usually reports archived entries, but an entry may be archived after the transaction simulation happened. |
+| `INVOKE_HOST_FUNCTION_TRAPPED`, diagnostics: HostError(Storage, LimitExceeded). | Host function tried to access a ledger entry outside of the footprint. Diagnostic events specify the exact missing entry. [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) should usually return the valid footprint, so in the case when [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) is being used, it’s likely that: a) Contract generates non-deterministic keys (e.g. derives them from RNG or ledger sequence), b) Contract has non-deterministic logic that results in access to a different entry set (e.g. a ‘lottery’ contract that non-deterministically performs transfer to either address A or address B). | If simualate transaction is not used, fix client code to build the correct footprint. If using simualate transaction, fix contract or footprint: avoid non-deterministic keys , and add all keys that can be accessed in any scenario, (e.g. balance entries for both A and B in the ‘lottery’ contract example). |
+| `INVOKE_HOST_FUNCTION_TRAPPED`, diagnostics: HostError(WasmVm, InvalidAction) | Failure in a Wasm contract, typically a `panic!()`. Diagnostic events might provide more detailed information, but not always, as the `panic!()` messages are not included in the Wasm builds. | Fix contract logic. Consider using `panic_with_error!()` and increase simualate transaction runs and unit testing. |
```

Comment:
typo on `simualate`. Also, noticed this usage of phrasing `simulate transaction`, instead of the proper noun, [`simulateTransaction`](../../../../docs/data/rpc/api-reference/methods/simulateTransaction) as used earlier in this same page to reference the process of simulating a transaction, it seems slightly inconsistent, for consideration, the other references could just be code formatted name, `simulateTransaction`, and not hyperlinked?